bin/compile_dependencies.sh

```diff
@@ -6,15 +6,21 @@ toplevel=$(git rev-parse --show-toplevel)
 
 cd $toplevel
 
+export CUSTOM_COMPILE_COMMAND="./bin/compile_dependencies.sh"
+
 # Base deps
 pip-compile \
-    --no-index \
+    --no-emit-index-url \
+    --allow-unsafe \
+    "$@" \
     requirements/base.in
 
 # Jenkins/tests deps
 pip-compile \
-    --no-index \
+    --no-emit-index-url \
+    --allow-unsafe \
     --output-file requirements/jenkins.txt \
+    "$@" \
     requirements/base.txt \
     requirements/testing.in \
```

Comment:
Zou je de requirements bestand structuur willen aanhouden zoals bijvoorbeeld in de [notificaties API](https://github.com/VNG-Realisatie/notificaties-api/tree/master/requirements)? Dit is overigens nog niet doorgevoerd in alle componenten.

Reply:
Dit is volgens mij nog niet helemaal hetzelfde als in de notificaties API, vergelijk beide anders nog even.